src/locale/sv.yml

```diff
@@ -1416,6 +1416,18 @@ feat:
       personRecordColumn: Svarande
       suggestedPeople: Föreslagna personer
       unlink: Avlänka
+    surveyForm:
```

Comment:
We generally don't change the YAML files manually. Translators provide them through https://translate.zetkin.org. But the fact that they're here made me want to test them, but when I set my browser language to Swedish the UI is still showing the English strings. Any ideas why?

Reply:
Untranslated new strings!

> Error: [@formatjs/intl Error MISSING_TRANSLATION] Missing message: "feat.surveys.surveyForm.submit" for locale "sv", using default message (Submit) as fallback.